testament/important_packages.nim

```diff
@@ -82,7 +82,7 @@ pkg "sdl2_nim", false, "nim c -r sdl2/sdl.nim"
 pkg "snip", false, "nimble test", "https://github.com/genotrance/snip"
 pkg "stint", false, "nim c -o:stintt -r stint.nim"
 pkg "strunicode", true, "nim c -r src/strunicode.nim"
-pkg "telebot", true, "nim c -o:tbot --oldgensym:on -r telebot.nim"
+pkg "telebot", true, "nim c -o:tbot -r src/telebot.nim"
```

Comment:
is `-o:tbot` needed? (simpler + conformity to other tests is better unless you have a good reason)

Reply:
im not sure, i didnt add this test

Reply:
ya I'm pretty sure it's safe to remove `-o:tbot` after looking at your files.

(ideally this would run `nimble test` if you had some tests but I'm not seeing any tests so this is fine)

Reply:
`-o:tbot` was needed before when there was `telebot` folder, which had caused confusion with `telebot` executable.